esp8266/modnetwork.c

```diff
@@ -36,9 +36,15 @@
 #include "queue.h"
 #include "user_interface.h"
 #include "espconn.h"
+#include "ip_addr.h"
 #include "spi_flash.h"
 #include "utils.h"
 
+#define WLAN_STA    0
+#define WLAN_AP     1
+#define WLAN_DHCP   "dhcp"
```

Comment:
This isn't needed, just have QSTR_dhcp and check it by qstr value, not by literal string value.